lib/ui/package.json

```diff
@@ -23,6 +23,8 @@
   },
   "dependencies": {
     "@storybook/addons": "5.2.0-beta.19",
+    "@storybook/addon-actions": "5.2.0-beta.19",
```

Comment:
why is `@storybook/ui` now installing some addons ? Shouldn't they be dev dependencies instead ?

Reply:
I'll check it out.

Reply:
You are correct, I'll issue a PR